test/dummy/app/views/components/block_alert.html.erb

```diff
@@ -0,0 +1,9 @@
+<%= render(OclTools::BlockAlertComponent.new) do |component| %>
+    <% component.with(:title) do %>
+        Safeguarding
```

Comment:
Can you use a generic example? e.g. link to google or github?

Reply:
done ;)